src/gs-shell-category.ui

```diff
@@ -127,15 +127,14 @@
 
                             <child>
                               <object class="GtkFlowBox" id="category_detail_box">
-                                <property name="margin_start">24</property>
-                                <property name="margin_end">24</property>
+                                <property name="margin_end">12</property>
                                 <property name="margin_top">21</property>
                                 <property name="margin_bottom">21</property>
                                 <property name="halign">center</property>
                                 <property name="visible">True</property>
                                 <property name="can_focus">False</property>
-                                <property name="row_spacing">20</property>
-                                <property name="column_spacing">20</property>
```

Comment:
These values came from Robin's design specs. After you decrease the margins above (which are also from the spec), does it still not fit in composite? (I think that the row spacing is more important for the UX than the margins so we should keep it if it works for us)